3.process_cfret_features/scripts/3.qc_report.r

```diff
@@ -0,0 +1,86 @@
+suppressPackageStartupMessages(library(dplyr))
+suppressPackageStartupMessages(library(ggplot2))
+suppressPackageStartupMessages(library(platetools))
+suppressPackageStartupMessages(library(stringr))
+suppressPackageStartupMessages(library(RColorBrewer))
+suppressPackageStartupMessages(library(arrow))
+
+# Paths to parquet files to generate QC report
+path_to_orig_plate4 <- file.path("./data/converted_profiles/localhost231120090001_converted.parquet")
+path_to_cleaned_plate4 <- file.path("./data/cleaned_profiles/localhost231120090001_cleaned.parquet")
+
+# Output path for bar chart
+orig_plate4_output_file <- file.path(paste0("./qc_figures/localhost231120090001_orig_plate4_platemap.png"))
+
+# Output path for platemap
+clean_plate4_output_file <- file.path(paste0("./qc_figures/localhost231120090001_cleaned_plate4_platemap.png"))
+
+# Read in CSV files
+orig_df <- arrow::read_parquet(path_to_orig_plate4)
+cleaned_df <- arrow::read_parquet(path_to_cleaned_plate4)
+
+# Assuming your dataframe is called 'orig_df'
```

Comment:
```suggestion
# Assuming your dataframe is called 'cleaned_df'
```

Reply:
Fixed this and the duplicate code. Thanks for catching this!